lib/chef/config.rb

```diff
@@ -319,7 +319,17 @@ def self.path_accessible?(path)
       default :host, 'localhost'
       default :port, 8889.upto(9999) # Will try ports from 8889-9999 until one works
     end
+
     default :chef_server_url,   "https://localhost:443"
+    default(:chef_server_root) do
+      # if the chef_server_url is a path to an organization, aka
+      # 'some_url.../organizations/*' then remove the '/organization/*' by default
+      if self.configuration[:chef_server_url] =~ /\/organizations\/\S*$/
```

Comment:
It would be nice to have a test for this. I tried it manually with both `https://chef.example/extra/root/organizations/myorg` and `https://chef.example/extra/root/organizations/myorg/` (without/with trailing slash) and it seemed to work okay, but if there happens to be a bug there it'd be nice to already have a place to add a regression test.